gnat2goto/driver/tree_walk.adb

```diff
@@ -2331,7 +2548,7 @@ package body Tree_Walk is
       RHS_Length : Irep;
       LHS_Copy : Irep;
       RHS_Copy : Irep;
-
+      Pre_Ret : Entity_Id := Etype (First_Index (Etype (N)));
```

Comment:
This is only used once way further down, might make sense to move it to a declare block instead.